R/01-getData.R

```diff
@@ -40,7 +40,7 @@ getFields <- function() {
 }
 
 #' Get list of databases available
-#'
+#' @return A list of possible database names
```

Comment:
Is this a 'list' or a 'character' ? Please check and verify with `str` and/or `class`.

Reply:
this is teh output:

["14CSea" ,  "IntChron" , "IsoArcH" , "LiVES"]

Reply:
please verify with `str`, but it appears that it is not a list, but a character vector with the names of the available databases

Reply:
yes, youre right it is a character vector